tests/PhpCollection/Tests/SequenceTest.php

```diff
@@ -178,7 +178,7 @@ public function testAdd()
             return 1;
         });
 
-        $this->assertSame(array(0, 0, 1, $this->a, $this->b), $this->seq->all());
+        $this->assertEquals(array(0, 0, 1, $this->a, $this->b), $this->seq->all());
```

Comment:
Could you tell me a bit more about this change? Looking at the failing test-case, it seems like the order of a and b changed which should not happen for a sequence type?

Reply:
Yeah, these hex IDs confused me, but sure they changed places. Let me see if I can find a reason.

Reply:
So what I think is happening is that `sortWith` uses `usort` which does not guarantee that elements come in specific order. And [there's this commit](https://github.com/php/php-src/commit/ace9fe53173974ec63f9e0357a6e8372ff5adb3b) which specifically mentions that usort may be affected, which commit went into PHP 7.0 release.

And indeed if I swap `$a` and `$b` in the input data, this specific assertion checks out all right.